packages/components/package.json

```diff
@@ -448,10 +448,6 @@
     "@rollup/plugin-commonjs": "^25.0.7",
     "@rollup/plugin-node-resolve": "15.2.3",
     "@rollup/plugin-typescript": "^11.1.6",
-    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
Changes from the actual bundled output of components is just file renames with different hashes and updates to imports to go the the renamed files